lib/sycamore/sycamore/transforms/standardizer.py

```diff
@@ -185,6 +185,9 @@ class DateTimeStandardizer(Standardizer):
     """
 
     DEFAULT_FORMAT = "%B %d, %Y %H:%M:%S%Z"
+    clock_re = re.compile(r"\d:[0-5]\d")
+    year_re = re.compile(r"([12]\d\d\d-)|(/[12]\d\d\d)|(\d/[0-3]?\d/\d)")
+    digitpair_re = re.compile(r"([0-2]\d)([0-5]\d)(\d\d)?")
```

Comment:
Would appreciate comments for the regexes

Reply:
I had hoped the names were descriptive.  Do you want examples of what they match?  Not sure how to make regexes clearer without just explaining the grammar of regexes, which is already better done on various web pages.

Reply:
Examples would be good, or even a short explanation of their functionality. It is very dense right now, and not easy to understand without trying to make sense of the regex itself.